File: Python/Assignments/9082017/algorithm.py
Class NBATeam(object):
	def __init__(self, name, city, color):
		self.name = name
		self.city = city
		self.color = color
		self.players = []

Class Player(object):
	def __init__(self, name, position, jerseyNumber):
		self.name = name
		self.position = position
		self.jerseyNumber = jerseyNumber

GSW = NBATeam("Golden State Warriors", "Oakland", ["blue", "gold"])
GSW.players.append(Player("Stephen Curry", "PG", 30))

File: Python/Assignments/9082017/algorithm.js
function NBATeam(name, city, color)
{
	this.name = name;
	this.city = city;
	this.color = color;
	this.players = [];
}

function Player(name, position, jerseyNumber)
{
	this.name = name;
	this.position = position;
	this.jerseyNumber = jerseyNumber;
}

var GSW = new NBATeam("Golden State Warriors", "Oakland", ["blue", "gold"]);
GSW.players.push(new Player("Stephen Curry", "PG", 30));